src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -597,6 +618,7 @@ protected override void OnNodePropertyChanged(object sender, PropertyChangedEven
             {
                 case "CachedValue":
                     Debug.WriteLine(string.Format("Requesting render packages for {0}", node.GUID));
```

Comment:
Now that we are removing geometry for node here, can we not remove geometry from GenerateViewGeometryFromRenderPackagesAndRequestUpdate method?